lms/djangoapps/branding/api.py

```diff
@@ -34,7 +35,7 @@ def is_enabled():
     return BrandingApiConfig.current().enabled
 
 
-def get_footer(is_secure=True):
+def get_footer(is_secure=True, language='en'):
```

Comment:
Use `settings.LANGUAGE_CODE` here and on all other places instead of hardcoding.